grails-app/utils/org/pih/warehouse/LocalizationUtil.groovy

```diff
@@ -148,4 +149,16 @@ class LocalizationUtil {
     static String getDefaultString(String str) {
         return getLocalizedString(str, null)
     }
+
+
+    static boolean isSpanishMexico(Locale locale) {
+        return locale.language == "es" && locale.country == "MX"
+    }
+
+    static String getLocalizedOrderImportDateFormat(Locale locale) {
+        if (isSpanishMexico(locale)) {
+            return Constants.EUROPEAN_DATE_FORMAT
```

Comment:
Perhaps it is worth having separate const for this purpose and not use the `EUROPEAN_DATE_FORMAT`?